shuup/admin/static_src/base/scss/shuup/auth.scss

```diff
@@ -15,6 +32,19 @@
             margin-bottom: 20px;
         }
     }
+    @include media-breakpoint-down(sm) {
+        margin-bottom: 15px;
+        margin-top: 5px;
+    }
+    @include media-breakpoint-up(sm) {
+        &.ipad {
+            padding-top: 20px;
+        }
+        &.android {
```

Comment:
We really can't make the admin login and logout work without checking which device is in use?

Reply:
It's arguably not necessary. If you remove those the menu wont be in the middle on all devices, it'll be more to the top on some and more to the bottom on other devices. I'm gonna finish the retractable menu issue soon so after that I'm gonna search for better solutions to this.

Reply:
I guess the real check is the screen height here.. if there is "almost" no space in vertical, remove margin/padding from the top